production/scrapers/louisiana_youth.R

```diff
@@ -0,0 +1,86 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+louisiana_youth_pull <- function(x){
+    xml2::read_html(x)
+}
+
+louisiana_youth_restruct <- function(x){
+    x %>%
+        rvest::html_node('.wp-block-table') %>%
+        rvest::html_table()
+}
+
+louisiana_youth_extract <- function(x){
+    la <- x %>%
+        janitor::row_to_names(row_number = 1) 
+    
+    check_names(la, 
+                c("Secure Care Facility", 
+                  "# of Youth Positive", 
+                  "# of Youth Recovered"))
+    
+    la_cln <- la %>%
+        select(Name = `Secure Care Facility`, 
+               Residents.Confirmed = `# of Youth Positive`,
+               Residents.Recovered = `# of Youth Recovered`) %>%
+        mutate(Name = str_c(toupper(Name), " YOUTH"))
+    
+    out <- la_cln %>%
+        clean_scraped_df() %>%
+        as_tibble() %>%
+        filter(Name != "TOTAL YOUTH") %>%
+    
+    return(out)
+}
+
+#' Scraper class for general Louisiana youth COVID data
+#' 
+#' @name louisiana_youth_scraper
+#' @description LA data self contained within html table. 
+#' \describe{
+#'   \item{Location}{The facilty name}
+#'   \item{Residents.Confirmed}{Residents Confirmed}
+#'   \item{Residents.Recovered}{Residents Recovered}
+#' }
```

Comment:
It looks like they also have a table on the same page with staff data (`Staff.Confirmed` and `Staff.Recovered`) incase you want to also pull those numbers!

Reply:
Just added that, thanks for catching!